README.md

```diff
@@ -145,17 +145,19 @@ make wasm for=transfer
 
 ## 🐳 Docker support
 
+### Local Ephemeral Node
+
 It's also possible to run a local ephemeral node with Docker.
 
 To build the Docker image with archive:
 ```bash
-docker build -t rusk .
+docker build -f Dockerfile.ephemeral -t rusk .
```

Comment:
The `-f` flag is for the name of the docker right? Why is it `Dockerfile.ephemeral`?

Reply:
Because it's for the ephemeral dev container. I changed it from `Dockerfile` to `Dockerfile.ephemeral` because there is now another dockerfile for containers with persistent.